akka-actor/build.sbt

```diff
@@ -5,5 +5,9 @@ Formatting.formatSettings
 OSGi.actor
 Dependencies.actor
 Version.versionSettings
+unmanagedSourceDirectories in Compile += {
+  val base = scalaVersion.value.take(4)
+  sourceDirectory.value / s"main-$base" / "scala"
+}
```

Comment:
isn't something like this supported by sbt out of the box?
http://stackoverflow.com/questions/31090822/what-are-the-scala-version-specific-source-directories-in-sbt